arcgis-ios-sdk-samples/Analysis/Analyze hotspots/AnalyzeHotspotsViewController.swift

```diff
@@ -21,9 +21,9 @@ class AnalyzeHotspotsViewController: UIViewController, HotspotSettingsVCDelegate
     @IBOutlet var containerView: UIView!
     
     
-    private var geoprocessingTask: AGSGeoprocessingTask!
-    private var geoprocessingJob: AGSGeoprocessingJob!
-    private var graphicsOverlay = AGSGraphicsOverlay()
+    fileprivate var geoprocessingTask: AGSGeoprocessingTask!
```

Comment:
It should be `private` instead of `fileprivate`. I made the change in macOS but missed out in iOS.